source/extensions/filters/http/cache/http_cache_utils.cc

```diff
@@ -95,7 +95,7 @@ SystemTime::duration eatLeadingDuration(absl::string_view& s) {
   }
   const absl::string_view digits(s.begin(), digits_length);
   s.remove_prefix(digits_length);
-  uint64_t num;
+  int64_t num;
```

Comment:
Using `uint64_t` caused `int64_t` overflows to return `zero()` duration instead of `max()`.